cellpack/autopack/ingredient/Ingredient.py

```diff
@@ -1468,6 +1471,12 @@ def perturbAxis(self, amplitude):
         return (x + dx, y + dy, z + dz)
 
     def transformPoints(self, trans, rot, points):
+        output = []
+        for point in points:
+            output.append(numpy.matmul(rot[0:3, 0:3], point) + trans)
+        return output
+
+    def transformPoints_mult(self, trans, rot, points):
```

Comment:
what's the difference between `transformPoints` and `transformPoints_mult`? it looks like they both are returning an array of x, y, z positions

Reply:
The numpy matrix multiplication approach is faster than the loop.

I copied over the older version of transformPoints to transformPoints_mult in case the numpy version broke something. I think it can be removed.